### PR TITLE
Working simulation and data gathering

### PR DESCRIPTION
**Pull request by BartoszF**

### CODE REVIEW

File: src/pages/GamePage/Gameplay/CarFactory.js [hunk 1]
```diff
@@ -3,89 +3,90 @@ import { collisionCategories } from "./Constants";
...
-//   phaser.matter.world.add(carComposite);
+  //   carBody.setCollisionCategory(collisionCategories["BODY"]);
+  //   carBody.setCollidesWith([
+  //     collisionCategories["WORLD"],
+  //     collisionCategories["BODY"]
+  //   ]);
+
+  //   var LFTireAttach = createAttachPoint(
+  //     LFTire.body,
+  //     carBody.body,
```
Comment (mpdev10) [inline src/pages/GamePage/Gameplay/CarFactory.js:50]: Number of lines with comments kinda confuse me whether they are significant or not - I suggest not commiting such wall of text

**Review by mpdev10 — APPROVED** [review]
Ok, however please try to avoid commiting wall of comments

File: src/pages/GamePage/Gameplay/CarFactory.js (hunk 1 above)
Comment (BartoszF) [reply src/pages/GamePage/Gameplay/CarFactory.js:50]: As said in conversation with Olaf - it probably will be useful in the future so I've left this here.

**Review by mpdev10 — APPROVED** [review]
.

File: src/pages/GamePage/Gameplay/CarFactory.js (hunk 1 above)
Comment (mpdev10) [reply src/pages/GamePage/Gameplay/CarFactory.js:50]: Understood

**Pull request merged**